runtime-light/stdlib/crypto/crypto-functions.cpp

```diff
@@ -115,3 +110,192 @@ task_t<int64_t> f$openssl_verify(const string &data, const string &signature, co
 
   co_return 1;
 }
+
+static bool ichar_equals(char a, char b) noexcept {
+  return std::tolower(static_cast<unsigned char>(a)) == std::tolower(static_cast<unsigned char>(b));
+}
+
+static std::optional<tl::CipherAlgorithm> parse_algorithm(const string &method) noexcept {
+  using namespace std::string_view_literals;
+  std::string_view method_sv{method.c_str(), method.size()};
+
+  if (std::ranges::equal(method_sv, "aes-128-cbc"sv, ichar_equals)) {
```

Comment:
It looks like you use constants like `aes-128-cbc` multiple times. Let's define something like:
`constexpr std::string_view AES_128_CBC = "aes-128-cbc";`

Reply:
+